repairnator/repairnator-pipeline/src/test/java/fr/inria/spirals/repairnator/process/step/paths/TestComputeClasspath.java

```diff
@@ -85,7 +85,7 @@ public void testComputeClasspath() throws IOException {
         ComputeClasspath computeClasspath = new ComputeClasspath(inspector, true);
 
         cloneStep.setNextStep(new CheckoutBuggyBuild(inspector, true))
-                .setNextStep(new TestProject(inspector))
+                .setNextStep(new BuildProject(inspector))
```

Comment:
I'm not sure: why did you keep this step here? We should not need it at all.

Reply:
This was what I thought too, but it didn't work. So I changed for one that normally takes less time to execute at least.

Reply:
Any idea why it did not work?

Reply:
The only reason I can think right now is because the folder "target" is not available without maven step, which is expected in `ComputeClasspath`.

Reply:
You're right because I put target/classes and target/test-classes. So we should change the test to create the folder before calling the step and it should work properly.